challenge2/styles.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    /* Colors */
+    --primary-color: #FCFCFC;
+    --complimentary-red: #EF4444;
+}
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+html {
+    font-size: 62.5%;
```

Comment:
Is this `font-size` at the `html` level needed?

We don't usually use percentages for font sizes, it's more common to use `rem` or `px` for them